charts/datadog/templates/_container-security-agent.yaml

```diff
@@ -38,6 +38,10 @@
     - name: DD_COMPLIANCE_CONFIG_XCCDF_ENABLED
       value:  {{ .Values.datadog.securityAgent.compliance.xccdf.enabled | quote }}
     {{- end }}
+    {{- if .Values.datadog.securityAgent.compliance.host_benchmarks.enabled }}
+    - name: DD_COMPLIANCE_CONFIG_HOST_BENCHMARKS_ENABLED
+      value:  {{ .Values.datadog.securityAgent.compliance.host_benchmarks.enabled | quote }}
```

Comment:
will there be any issue if both the deprecated and new envvars are set, and they are in conflict?

Reply:
In the Agent, the `xccdf` feature will be enabled if either `xccdf.enabled` or `host_benchmarks.enabled` is true.
See https://github.com/DataDog/datadog-agent/blob/e400b455f725d0887af9e6f08fa10afad2aa3da1/pkg/compliance/agent.go#L77
I think it makes sense this way.

Reply:
I agree.